iocore/eventsystem/UnixEvent.cc

```diff
@@ -104,3 +106,19 @@ Event::schedule_every(ink_hrtime aperiod, int acallback_event)
     ethread->EventQueueExternal.enqueue_local(this);
   }
 }
+
+#ifdef ENABLE_EVENT_TRACKER
+
+void
+Event::set_location()
+{
+  _location = ink_backtrace(3);
```

Comment:
This is doing pretty tricky. Assuming this is called by `schedule_*` functions and records where they're called (the 3rd frame of the call stack).

Reply:
Any idea how performant this is? I.e. is it going to significantly impact how much traffic a production cache can take, and/or TTMS? Or is it very small, could we just leave this on in Production?

Reply:
> the 3rd frame of the call stack
Yeah, the whole "3" thing is awkward, but that's pretty common when looking back on the stack. It seems fragile, but it's also unlikely to change. Meh.

Reply:
I though `backtrace(3)` doesn't have noticeable overhead. Because it just refers address of current call stack. I'll measure on/off at production.
https://man7.org/linux/man-pages/man3/backtrace.3.html

Reply:
Ah, excellent. I was just curious. It's still very useful in debug mode, but there's so much more we can do if we can just leave it on all the time.